js/trek.js

```diff
@@ -0,0 +1,85 @@
+$( document ).ready(function() {
+  const loadTrips = function loadTrips() {
+    $.get('https://trektravel.herokuapp.com/trips',
+      response => {
+        response.forEach(function(trip) {
+          let tripInfo = `<li><p data-id=${trip.id}><strong>${trip.name}</strong> in ${trip.continent}</p></li>`
+          $('ul').append(tripInfo);
+        }); // end of forEach
+
+      }) // end of $.get
+      .fail(function(response){
+          console.log(response);
+          $('#fail').html('<p>Request was unsuccessful</p>')
+        }) // end .fail
+        .always(function(){
+          console.log('always even if we have success or failure');
+        });
+  }; // end of loadTrips()
+
+  const loadTrip = function loadTrip(id) {
+    $.get(`https://trektravel.herokuapp.com/trips/${id}`,
+      response => {
+        let tripInfo = `
```

Comment:
This callback ends up being quite long. It would probably be worthwhile to break some of the components out into separate named functions defined at the top of the file.